robot/Cumulus/resources/locators_47.py

```diff
@@ -170,7 +171,8 @@
         },
     'data_imports':{
         'status':'//div[contains(@class,"slds-tile__title")][./p[text()="BDI_DataImport_BATCH"]]/div[contains(@class,"slds-col")]/span[text()="{}"]',
-        'checkbox':'//tr[./th//a[@title="{}"]]/td//span[@class="slds-checkbox--faux"]'
+        'checkbox':'//tr[./th//a[@title="{}"]]/td//span[@class="slds-checkbox--faux"]',
+        'actions_dd':'//a[contains(@title,"more actions")and @aria-expanded="true"]',
```

Comment:
what does actions_dd refer to?

Reply:
actions dropdown, short form I use for dropdown is dd